src/mscorlib/src/System/Collections/CollectionBase.cs

```diff
@@ -181,33 +166,33 @@ public abstract class CollectionBase : IList {
             return InnerList.GetEnumerator();
         }
 
-        protected virtual void OnSet(int index, Object oldValue, Object newValue) { 
```

Comment:
I'm not sure why all of these protecteds are being changed to protected internals.  If this is what the rewriter is doing, seems like a bug in the rewriter.  protected internal is more accessible than protected, not less, as protected internal is an or, not an and.